lib/filesystem/src/filesystem.cpp

```diff
@@ -104,18 +104,22 @@ namespace irods::experimental::filesystem::NAMESPACE_IMPL
             // clang-format off
             if (_perm == "read_metadata")   { return perms::read_metadata; }
             if (_perm == "read_object")     { return perms::read; }
+            if (_perm == "read object")     { return perms::read; }
             if (_perm == "read")            { return perms::read; }
             if (_perm == "create_metadata") { return perms::create_metadata; }
             if (_perm == "modify_metadata") { return perms::modify_metadata; }
             if (_perm == "delete_metadata") { return perms::delete_metadata; }
             if (_perm == "create_object")   { return perms::create_object; }
             if (_perm == "modify_object")   { return perms::write; }
+            if (_perm == "modify object")   { return perms::write; }
             if (_perm == "write")           { return perms::write; }
             if (_perm == "delete_object")   { return perms::delete_object; }
             if (_perm == "own")             { return perms::own; }
+            if (_perm == "null")            { return perms::null; }
```

Comment:
consider putting 'null' at the 'top' of the list.

Reply:
Done